dpnp/dpnp_container.py

```diff
@@ -200,6 +203,22 @@ def ones(shape,
     return dpnp_array(array_obj.shape, buffer=array_obj, order=order)
 
 
+def tril(x1, /, *, k=0):
+    k = operator.index(k)
+    order = "F" if (x1.flags.f_contiguous) else "C"
```

Comment:
Shouldn't we use just:
```
return dpnp_array(array_obj.shape, buffer=array_obj, order="K")
```
will it guarantee the resulting usm_ndarray of the same order as passed by `buffer` here?